doc/source/whatsnew/v0.20.0.txt

```diff
@@ -43,6 +43,16 @@ fixed-width text files, and :func:`read_excel` for parsing Excel files.
    pd.read_fwf(StringIO(data)).dtypes
    pd.read_fwf(StringIO(data), dtype={'a':'float64', 'b':'object'}).dtypes
 
+You can now pass a dictionary mapping column names to desired data types for that
+column to :meth:`~DataFrame.astype`.
+
+.. ipython:: python
+
+   dft = pd.DataFrame({'a': [1,2,3], 'b': [4,5,6], 'c': [7, 8, 9]})
+   dft = dft.astype({'a': np.float64, 'c': np.uint8})
+   dft
+   dft.dtypes
+
```

Comment:
@m-charlton This is not needed, as this enhancement was already added in 0.19.0, and is mentioned in the whatsnew of 0.19.0 (see the changes in the PR https://github.com/pandas-dev/pandas/pull/13375/files)